ecs_fargate/assets/dashboards/amazon_fargate_overview.json

```diff
@@ -1,9 +1,9 @@
 {
     "author_info": {
         "author_name": "Datadog"
-    }, 
-    "board_title": "Amazon Fargate Overview", 
-    "created": "2020-01-22T22:21:55.684065+00:00", 
+    },
+    "board_title": "Amazon ECS on AWS Fargate - Overview",
```

Comment:
I think changing the board_title after it's been synced will cause issues. Would it be worth moving this into the description?